herbert_core/admin/is_jenkins.m

```diff
@@ -0,0 +1,16 @@
+function [is_jenkins_pc] = is_jenkins()
+% Attempt to determine if the PC we're running on is a Jenkins server.
+%   This is detected through environment variables that are always set by
+%   Jenkins.
```

Comment:
if they are set (presumably by something else) do you need to check all three, unless you're using them?

Reply:
They're not really used no, I guess I was just erring on the side of caution and using 3 values that are all definitely set. Just in case a user accidentally had one set. I can prune it down if you'd like?

Reply:
it's fine